docs/website/blog/2022-05-02-odo-v3-alpha1.md

```diff
@@ -0,0 +1,327 @@
+
+```
+? Select language: javascript
+? Select project type: Node.js Runtime
+ ✓  Downloading devfile "nodejs" from registry "DefaultDevfileRegistry" [961ms]
+Current component configuration:
+Container "runtime":
+  Opened ports:
+   - 3000
+  Environment variables:
+? Select container for which you want to change configuration? NONE - configuration is correct
+? Enter component name: my-nodejs-app
+
+Your new component 'my-nodejs-app' is ready in the current directory.
+To start editing your component, use 'odo dev' and open this folder in your favorite IDE.
+Changes will be directly reflected on the cluster.
+```
+
+Finally, if you start the `odo init` command from an empty directory, it will give you the choice to download one of the starter projects listed in the Devfile.
+
+## The development phase
+
+Now that a Devfile is present in the current directory, you can run your application in development mode, using the `odo dev` command. This command will create a `Deployment` in the cluster that will help start a container as defined in the Devfile. Then, the sources present in the current directory will be synchronized into the container, and the commands to build and run the application will be executed from inside the container.
+
+At the same time, a port-forwarding will be done for each endpoint defined in the Devfile, so you can access the container ports through local ports in your development machine.
+
+Finally, `odo` will watch for changes in the current directory. When files are modified, added or deleted, `odo` will synchronize the changes to the container, and will restart the build and run commands from inside the container.
+
+```
+$ odo dev
+  __
+ /  \__     Developing using the my-nodejs-app Devfile
+ \__/  \    Namespace: prj2
+ /  \__/    odo version: v3.0.0-alpha1
+ \__/
+
+↪ Deploying to the cluster in developer mode
+ ✓  Waiting for Kubernetes resources [6s]
+ ✓  Syncing files into the container [439ms]
+ ✓  Building your application in container on cluster [3s]
+ ✓  Executing the application [1s]
+
+Your application is now running on the cluster
+ - Forwarding from 127.0.0.1:40001 -> 3000
+ - Forwarding from 127.0.0.1:40002 -> 5858
+
+Watching for changes in the current directory /home/phmartin/Documents/tests/devto-deploy
+Press Ctrl+c to exit `odo dev` and delete resources from the cluster
+```
+
+To be able to debug the application, you will need to run the `odo dev --debug` command instead.
+
+When you have finished the development session, you just need to hit Ctrl-c to stop the `odo dev` command. The command won't terminate immediately, as it will delete the resources it has deployed on the cluster before to exit.
```

Comment:
```suggestion
When you have finished the development session, you just need to hit Ctrl-c to stop the `odo dev` command. The command won't terminate immediately, as it will delete the resources it has deployed on the cluster before exiting.
```